psvpfsparser/PsvPfsParserConfig.cpp

```diff
@@ -108,9 +108,6 @@ int parse_options(int argc, char* argv[], PsvPfsParserConfig& cfg)
 
       if(!f00d_url.empty())
```

Comment:
I agree with this commit. It is good that you did not remove url option.
I have made them hidden, but they are left for backwards compatibility.
Curl related code could be indeed removed.